BUILD.md

```diff
@@ -1,12 +1,24 @@
+#### Normalista uses [Github Pages](https://help.github.com/articles/using-jekyll-as-a-static-site-generator-with-github-pages/) to make easy to mantain and publish academic content.
+
+The stack is based on gulp and jekyll.
+
 ## Installation
 
-Install jekyll following http://jekyllrb.com/docs/installation/
+Install [jekyll](http://jekyllrb.com/docs/installation/).
+
+Install [npm](https://docs.npmjs.com/getting-started/installing-node) via node.
```

Comment:
So the requirement would be to have node.js installed. Node ships with npm. Most people know if they have node, but not if they have npm. Saying that "node.js" is the requirement will make more sense. Also it's easier to just go and install node in any platform